src/app/admin/admin-reports/filters-section/filters-section.component.ts

```diff
@@ -0,0 +1,145 @@
+import { Component, Input } from '@angular/core';
+import { FormBuilder, FormControl, FormGroup } from '@angular/forms';
+import { FilterGroup } from './filter-group.model';
+import { Filter } from './filter.model';
+
+@Component({
+  selector: 'ds-filters',
+  templateUrl: './filters-section.component.html',
+  styleUrls: ['./filters-section.component.scss']
+})
+export class FiltersComponent {
```

Comment:
Again, it's good to add a comment to briefly describe every component's usage

Reply:
Done.